### PR TITLE
Move relay-runtime to peerDeps

### PR DESCRIPTION
**Pull request by Nodonisko**
Without this, won't work with relay 1.5+. I think this is better way to provide compatibility across different relay versions.

More info: https://github.com/facebook/relay/issues/2369

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,12 +32,10 @@
     "url": "https://github.com/relay-tools/relay-local-schema/issues"
   },
   "homepage": "https://github.com/relay-tools/relay-local-schema#readme",
-  "dependencies": {
-    "relay-runtime": "^1.0.0"
-  },
   "peerDependencies": {
     "graphql": ">=0.5.0",
-    "react-relay": ">=0.8.0"
+    "react-relay": ">=0.8.0",
```
Comment (taion) [inline package.json]: heh, this should probably be `>=1.0.0`

File: package.json [hunk 2]
```diff
@@ -32,12 +32,10 @@
...
+    "relay-runtime": ">=1.4.0"
```
Comment (taion) [inline package.json]: this can be `>=1.0.0` as well?

**Comment by Nodonisko** [comment]
Updated to `>=1.0.0`

**Comment by taion** [comment]
Wait, no, this isn't right – if it's just a renamed function, we should just bump the dependency and use the right name.

**Comment by taion** [comment]
Hmm, never mind.

**Comment by taion** [comment]
Released in v0.7.0.

**Pull request merged**